scipy/linalg/matfuncs.py

```diff
@@ -668,3 +669,56 @@ def rounded_sign(x):
         return S0
     else:
         return S0, errest
+
+
+def khatri_rao(a, b):
+    """
```

Comment:
Make this `r"""` such that `\` in the math environment doesn't cause issues in the refguide_check

Reply:
@ilayn what exactly do you mean?

Reply:
Instead of `"""` put an r in the front making it `r"""`

Reply:
done